webrtc/interfaces.https.html

```diff
@@ -107,13 +107,20 @@
       }));
   }
 
-  // Main function to do the IDL test, using fetched IDL text
-  function doIdlTest(idlText) {
+  promise_test(async t => {
+    await asyncInit();
+
     const idlArray = new IdlArray();
 
+    let webrtcIdl = fetch('/interfaces/webrtc-pc.idl').then(r => r.text());
+    let mediacaptureMainIdl = fetch('/interfaces/mediacapture-main.idl').then(r => r.text());
+
+    idlArray.add_untested_idls(mediacaptureMainIdl, { only: ['MediaStreamConstraints'] });
+    idlArray.add_idls(webrtcIdl);
+
     idlArray.add_untested_idls('interface EventHandler {};');
+    idlArray.add_idls('interface EventTarget {};');
```

Comment:
This is is example missing interface which had poor error messages (https://github.com/w3c/web-platform-tests/issues/10214)